metapool/tests/test_metapool.py

```diff
@@ -924,6 +926,13 @@ def test_calculate_iseqnorm_pooling_volumes(self):
         exp = pd.read_csv(exp_fp, sep=',')
         pd.testing.assert_frame_equal(obs, exp)
 
+        # TEST NAN IN ORIGINAL DATA ARE HANDLED CORRECTLY
+        obs = calculate_iseqnorm_pooling_volumes(self.with_nan)
+
+        # if this cell's value is NaN, then calculate_iseqnorm_pooling_volumes
+        # is not handling NaN values properly.
+        self.assertFalse(pd.isna(obs['Filtered Reads'][3]))
```

Comment:
the NaN will be overwritten by the max(LoadingFactor), which from the test data I think should be 3.5(?)

Nonetheless, thanks for including a nan input test. Evidently, it was needed.